crates/common_utils/src/events.rs

```diff
@@ -124,10 +127,6 @@ impl_api_event_type!(
     (
         String,
         id_type::MerchantId,
-        (id_type::MerchantId, String),
-        (id_type::MerchantId, &String),
-        (&id_type::MerchantId, &String),
-        (&String, &String),
```

Comment:
Can you please give context on why these are being removed

Reply:
```Rust
(id_type::MerchantId, String),
(&id_type::MerchantId, &String),
```

These 2 were used in `crates/router/src/routes/api_keys.rs`

The `String` has been replaced by the `ApiKeyId`. The impl has been moved to `crates/common_utils/src/id_type/api_key.rs`


While removing them I noticed that these:

```Rust
        (id_type::MerchantId, &String),
        (&String, &String),
```

seemed unused as well, so removed them.